server/src/models/Expense.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+
+# Location is stored as an x and y coordinate (easier to input into maps APIs)
+# All fields are currently required
+class Expense(BaseModel):
+    id: int
+    name: str
+    description: str
+    date: date
+    price: float
+    currency: str 
+    exchange_rate: float 
+    locationx: float
+    locationy: float 
```

Comment:
For now I would keep a location string field since the user should be able to just type the name of a place for example.
```suggestion
    location: Optional[str] = None
    locationx: Optional[float] = None
    locationy: Optiona[float] = None
```